src/domains/domain.h

```diff
@@ -54,6 +60,11 @@ class domaint
 
   typedef std::vector<var_spect> var_specst;
 
+  // handles on values to retrieve from model
```

Comment:
Why is this required as part of the domain interface?

Reply:
Mostly because all functionality that uses it moved into domain and keeping it in solver would just mean passing references there and back. (and possibly also some better reason that I cannot now remember why I dropped it here)